src/pog.gleam

```diff
@@ -57,6 +57,20 @@ pub type Config {
   )
 }
 
+pub type Ssl {
+  /// Enable SSL connection, but don't check CA certificate.
+  /// `SslEnabled` should always be prioritized upon `SslUnsafe`.
+  /// As it implies, that option is unsafe, so you should use this option only
+  /// if you know what you're doing. In case `pog` can not find the proper CA
+  /// certificate, take a look at the README to get some help to inject the CA
+  /// certificate in your OS.
+  SslUnsafe
+  /// Enable SSL connection, and check CA certificate.
+  SslEnabled
+  /// Disable SSL connection.
+  SslDisabled
+}
```

Comment:
Bike shedding on names here!

It kinda sounds like "disabled" is more safe than "unsafe", but I would have thought that enabled > verified but unchecked > unverified. Is that right? Could the names get scarier and uglier as they get less secure?

Reply:
The idea is `SslEnabled` > `SslUnsafe` > `SslDisabled`. Maybe we can use `SslVerified` > `SslUnverified` > `SslDisabled`?

Reply:
Implemented this.